addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -37,30 +37,54 @@ def _get_wizard_values(self):
     # COMPUTE/CONSTRAINS METHODS
     # -------------------------------------------------------------------------
 
+    @api.depends('l10n_it_edi_actionable_errors')
+    def _compute_l10n_it_edi_warning_message(self):
+        # To be removed -- Proxy feature to be replaced with actionable_errors as soon as the user updates the module
+        for wizard in self:
+            messages = []
+            if wizard.l10n_it_edi_actionable_errors:
+                messages.append(_("Please upgrade the Italian EDI module to update this widget."))
+                messages.append(_("Go to Applications page and update the 'Italia - Fatturazione Elettronica' module."))
+                messages.append("")
+                for error_key, error_data in wizard.l10n_it_edi_actionable_errors.items():
+                    message = error_data['message']
+                    split = error_key.split("_")
+                    if len(split) > 1 and (model_id := {
+                        'partner': 'res.partner',
+                        'move': 'account.move',
+                        'company': 'res.company'
+                    }.get(split[0], None)):
+                        action = error_data['action']
+                        if 'res_id' in action:
+                            record_ids = [action['res_id']]
+                        else:
+                            record_ids = action['domain'][0][2]
+                        records = self.env[model_id].browse(record_ids)
+                        message = f"{message} - {', '.join(records.mapped('display_name'))}"
+                    messages.append(nl2br(message))
```

Comment:
That one won't work without an explicit conversion to markup:
```py
>>> messages = ["Hello!", nl2br("Hello\nWorld!")]
>>> messages
['Hello!', 'Hello<br>\nWorld!']
>>> Markup("<br/>").join(messages)
Markup('Hello!<br/>Hello&lt;br&gt;\nWorld!')
```
```suggestion
                    messages.append(nl2br(escape(message)))
```